main.go

```diff
@@ -149,6 +162,12 @@ var cmdInstall = &cli.Command{
 
 		i := lib.NewInstall(vers, c.Bool("no-check"), c.Bool("allow-downgrade"), fetcher)
 		err := i.Run(c.Context)
+		for retries := c.Int("retries"); retries > 0; retries-- {
+			if err == nil {
+				break
+			}
+			err = i.Run(c.Context)
```

Comment:
We definitely shouldn't try installing multiple times. If something goes wrong when actually _installing_, something is probably very wrong.

Reply:
You're probably right, I'll look into that as well